ui/components/app/wallet-overview/index.scss

```diff
@@ -172,3 +172,14 @@
     margin-top: -4px;
   }
 }
+
+.coin-buttons-toast {
```

Comment:
This is a bit of a wide toast in fullscreen

![Screenshot 2024-12-18 at 11 01 05](https://github.com/user-attachments/assets/a313d246-0aed-4a0d-8bdb-a9d488758e00)

Should we add a max width to this toast?

Reply:
https://www.loom.com/share/4d69acc7a295487f909d2fff582d57e1?sid=a386edf5-75cb-42d0-9fa0-233e3b173b68

Reply:
Hmm good call. Let me see if I can get this to render more nicely on full screen 👍

Reply:
<img width="1913" alt="Screenshot 2024-12-18 at 9 24 50 AM" src="https://github.com/user-attachments/assets/a9ceea79-122d-4351-8a3e-b9fd1000507c" />

After poking around, it seems like this is the existing behavior. As we chatted about on Slack, maybe we can talk to design tokens team to update.